js/packages/openinference-instrumentation-langchain/src/instrumentation.ts

```diff
@@ -31,12 +32,20 @@ type CallbackManagerModule =
   | typeof CallbackManagerModuleV02;
 
 export class LangChainInstrumentation extends InstrumentationBase<CallbackManagerModule> {
-  constructor(config?: InstrumentationConfig) {
+  private oiTracer: OITracer;
+
+  constructor(
+    config?: InstrumentationConfig & { traceConfig?: TraceConfigOptions },
```

Comment:
this is probably fine, for openai iu put them under two different keys to be less confusing, maybe it's more though? Also i added doc comment to try and help